### DISCUSSION TITLE
relative link to html inserts "https..."

### DISCUSSION THREAD

**Discussion by chipbrock** [executablebooks/meta#408] [Q&A]
Working in Jupyterbooks converting from working RStudio and bookdown.

I am trying to point to an html page that is in the same directory set as my primary html page. I do this all the time in bookdown and RStudio. 

Two tries.

First: from Bookdown....Inside of my page I would insert:

[TITLE1out](./examples/relativity/dragon_1_E3.html){target="_blank"}

and would get <a href="./examples/relativity/dragon_1_E3.html" target="_blank">TITLE1out</a>

great.

Second: from Jupyterbook I insert:

[TITLE1out](./examples/relativity/dragon_1_E3.html){target="_blank"}

But what's rendered in the processed html from Jupyterbooks (PyCharm) is 

<p><span class="xref myst">TITLE1out</span></p>

This seems strange behavior to me and not helpful. I'm very used to looking at the rendered html locally before serving it up to my host.

Any advice would be appreciated because this is a killer for me in this conversion. I've got dozens of these kinds of extraneous pages of content.

**Comment by chrisjsewell** [comment]
`[TITLE1out](./examples/relativity/dragon_1_E3.html){target="_blank"}` is not a supported syntax, that’s very rmarkdown specific

When you say you are trying to point another html page, is this page originally a markdown/notebook page that I’ve converted to html? Or are using directly HTML pages?

**Reply by chipbrock** [reply]
I apologize. I think I figured it out... I need to do 

<a href="../examples/relativity/dragon_1_E3.html">TITLE3out</a>

I didn't report this but I think I was counting diretories up wrong. 

Thanks for jumping in...  I will delete this in a minute.

**Reply by chrisjsewell** [reply]
No worries 👍
